### ISSUE TITLE
Polygon archive node failed to start

### ISSUE THREAD

**Issue by sundafa** [ledgerwatch/erigon#7721]
#### System information

Erigon version: v2.45.2
OS & Version: Linux
Commit hash: 
Erigon Command (with flags/config):
Concensus Layer:
Concensus Layer Command (with flags/config):
Chain/Network: bor-mainnet

#### Expected behaviour
Polygon node starts and runs normally

#### Actual behaviour

run cmd:
`/node/archive/polygon/bin/erigon --datadir=/node/archive/polygon/erigon/ --chain=bor-mainnet --prune=disabled --bor.heimdall='http://ip:port' `

The node cannot be synchronized, and the log is always printed
`Span with high enough block number is not loaded fetching span=2028`

And the software version cannot be reduced
`migrator.VerifyVersion: cannot downgrade minor DB version from 6.1 to 6.6`

#### Backtrace

````
[backtrace]
````
![image](https://github.com/ledgerwatch/erigon/assets/24621599/4e0d286c-257e-43ec-b238-db3e3c497e2b)

**Comment by AskAlexSharov** [comment]
do you trying to downgrade erigon's version?

**Comment by AskAlexSharov** [comment]
`cannot downgrade minor DB version from 6.1 to 6.6` - it's mistake in logging. erigon mean `from 6.1 to 6.0`. 
it means you did you newer version of erigon, but then switch to older one

**Comment by sundafa** [comment]
No, tried to rollback the version, but failed

**Comment by AskAlexSharov** [comment]
Newer version has DB migration. We don't allow rollback in this case.

**Comment by sundafa** [comment]
It seems that you misunderstood my point. The point is not to roll back the version, but that the new version cannot run the Polygon Archive node.

**Comment by YuXiaoCoder** [comment]
I have the same problem，I run the service with the following command
```bash
/opt/maticmain/core/erigon --snapshots --chain=bor-mainnet --bor.heimdall=http://127.0.0.1:1317 --datadir=/mnt/maticmain/node/erigon --http --http.addr=0.0.0.0 --http.port=8545 --http.vhosts=* --http.api=admin,eth,erigon,net,trace,web3,debug,txpool,bor --http.trace --port=30303 --rpc.gascap=5000000000 --db.size.limit=10T --db.pagesize=16kb
```

The error log is as follows
```bash
INFO[06-15|10:26:52.835] logging to file system                   log dir=/mnt/maticmain/node/erigon/logs file prefix=erigon log level=info json=false
INFO[06-15|10:26:52.836] Build info                               git_branch=HEAD git_tag=v2.45.2 git_commit=048fe49e4f379829e92116326ed8fcec3c9c8c9a
INFO[06-15|10:26:52.836] Starting Erigon on Bor Mainnet...
INFO[06-15|10:26:52.838] Maximum peer count                       ETH=100 total=100
INFO[06-15|10:26:52.838] starting HTTP APIs                       APIs=admin,eth,erigon,net,trace,web3,debug,txpool,bor
INFO[06-15|10:26:52.838] torrent verbosity                        level=WRN
INFO[06-15|10:26:54.941] Set global gas cap                       cap=5000000000
INFO[06-15|10:26:54.942] [Downloader] Runnning with               ipv6-enabled=true ipv4-enabled=true download.rate=16mb upload.rate=4mb
INFO[06-15|10:26:54.943] Opening Database                         label=chaindata path=/mnt/maticmain/node/erigon/chaindata
mdbx_setup_dxb:15897 could not apply preconfigured db-geometry: err -30417
EROR[06-15|10:26:54.946] Erigon startup                           err="mdbx_env_open: MDBX_TOO_LARGE: Database is too large for current system, e.g. could NOT be mapped into RAM, label: chaindata, trace: [kv_mdbx.go:262 node.go:327 node.go:330 backend.go:208 node.go:105 main.go:66 make_app.go:42 command.go:274 app.go:332 app.go:309 main.go:37 proc.go:250 asm_amd64.s:1598]"
mdbx_env_open: MDBX_TOO_LARGE: Database is too large for current system, e.g. could NOT be mapped into RAM, label: chaindata, trace: [kv_mdbx.go:262 node.go:327 node.go:330 backend.go:208 node.go:105 main.go:66 make_app.go:42 command.go:274 app.go:332 app.go:309 main.go:37 proc.go:250 asm_amd64.s:1598]
```

**Comment by AskAlexSharov** [comment]
Try set explicit: --db.size.limit

**Comment by AskAlexSharov** [comment]
@YuXiaoCoder  try set  --db.size.limit=7T
likely your db has 4Kb pagesize - in this case can't go > 8Tb

**Comment by YuXiaoCoder** [comment]
@AskAlexSharov 
Your guess is correct, I downloaded the snapshot from https://snapshots.polygon.technology/, how can I change the size of Pagesize, is there a detailed command?
```bash
./mdbx_stat -ef /mnt/maticmain/node/erigon/chaindata/
mdbx_stat v0.12.0-71-g1cac6536 (2022-07-28T09:57:31+07:00, T-9a6d7e5b917e5fbd14dc51835fa749d092aa1d72)
Running for /mnt/maticmain/node/erigon/chaindata/...
Environment Info
  Pagesize: 4096
  Dynamic datafile: 49152..7696581394432 bytes (+2147483648/-4294967296), 12..1879048192 pages (+524288/-1048576)
  Current mapsize: 7696581394432 bytes, 1879048192 pages
  Current datafile: 6270652252160 bytes, 1530920960 pages
  Last transaction ID: 62020705
  Latter reader transaction ID: 62020705 (0)
  Max readers: 116
  Number of reader slots uses: 1
Garbage Collection
  Pagesize: 4096
  Tree depth: 2
  Branch pages: 1
  Leaf pages: 51
  Overflow pages: 7683
  Entries: 7711
Page Usage
  Total: 1879048192 100%
  Backed: 1530920960 81.5%
  Allocated: 1530606749 81.5%
  Remained: 348441443 18.5%
  Used: 1522782739 81.0%
  GC: 7824010 0.4%
  Retained: 270 0.0%
  Reclaimable: 7823740 0.4%
  Available: 356265183 19.0%
Status of Main DB
  Pagesize: 4096
  Tree depth: 2
  Branch pages: 1
  Leaf pages: 3
  Overflow pages: 0
  Entries: 111
```

**Comment by AskAlexSharov** [comment]
only by copy data to new db.
- create new db in another datadir (with new cli flags)
- copy data by `integration mdbx_to_mdbx`
